src/mixins/placeholder.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+
+/**
+ * CSS to style the selection psuedo-element.
+ *
+ * @example
+ * // Styles as object usage
+ * const styles = {
+ *   ...placeholder(styles)
+ * }
+ *
+ * // styled-components usage
+ * const div = styled.input`
+ *   ...placeholder(styles)
```

Comment:
No `...`, this should be `${placeholder(css`background: red`)} `

Since `css` returns an Array, you'll also need to amend the types to allow for that!